shared/src/business/useCaseHelper/initialFilingDocuments/updateInitialFilingDocuments.js

```diff
@@ -92,9 +92,10 @@ exports.updateInitialFilingDocuments = async ({
   caseEntity,
   caseToUpdate,
 }) => {
+  const PETITION_KEY = 'petitionFile';
   const initialDocumentTypesWithoutPetition = omit(
     INITIAL_DOCUMENT_TYPES_MAP,
-    'petitionFile',
+    PETITION_KEY,
```

Comment:
I'm willing to remove this change. I thought the name would add a little more meaning to why the string is what it is.